src/components/EntityInfo/EntityHeader.vue

```diff
@@ -120,8 +125,15 @@ export default class EntityHeader extends Vue {
     }
   }
 
-  /** The incorporation application or registration description. */
-  get iaRegDescription (): string {
+  /** The incorporation/registration/amalgamation application description. */
+  get appDescription (): string {
+    if (this.isDraftAmalgamation) {
+      return this.getTasks[0]?.task.filing.displayName
+    }
+    if (this.isFiledAmalgamation) {
+      return this.getFilings[0]?.displayName
+    }
```

Comment:
We get the display name from the first-and-only-task (in Todo List) or the first-and-only-filing (in Filing History List). This was the cleanest way I could think of doing this without storing other data that would be used only for this scenario.

This is kind of a special case because a filing = a draft business in this scenario.

Reply:
Sample named amalgamation:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/360a7421-7e16-4e51-bae0-ead2f479eaf5)

Sample numbered amalgamation:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/6c171179-db1a-4e76-a187-5b2d214a7224)